tests/test_circuit_graph.py

```diff
@@ -166,7 +166,7 @@ def test_op_indices(self, circuit):
     def test_layers(self, parameterized_circuit):
         """A test of a simple circuit with 3 layers and 6 parameters"""
 
-        dev = qml.device("default.qubit", wires=3)
+        dev = qml.device("default.gaussian", wires=3)
```

Comment:
why changed?

Reply:
`parameterized_circuit` is a CV QNode, I think this was a bug/typo.

Reply:
lol and it still passed before?

Reply:
Weirdly, yes. It was failing for me on this branch, because of the two line decomposition logic change in the QNode.